### DISCUSSION TITLE
What's the difference between metadata in Next.js and next-seo?

### DISCUSSION THREAD

**Discussion by snowriceDD** [vercel/next.js#51392] [Help] [answered]
### Summary

For search engine optimization and performance .

**Comment by icyJoseph** [comment, marked as answer]
Mmm I think in a way, metadata in the App Dir is a, next native API, aimed to replace `next-seo`, mostly because the Server Components Streaming, makes it a bit tricky to target the `head` of the document, and this is best handled from within Next.js itself.

The `next-seo`, is still the kind, I'd say, for Pages Dir.
